Makefile

```diff
@@ -30,7 +30,7 @@ BIN_DIR=$(shell pwd)/bin
 SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
-CONTAINER_ENGINE ?= podman
+CONTAINER_ENGINE ?= docker
```

Comment:
Does the security review require this change?

Reply:
Sorry, well spotted, I'll change it back